### ISSUE TITLE
Random 404 page + game.

### ISSUE THREAD

**Issue by Carreau** [jupyter/nbviewer#509]
Just cross my  mind, 

it would be nice to have nice Error page in general, possibly with random images, or small JS game, 
like  snake which eats various language logo and grows.

That would be  nice project for someone new.

**Comment by willingc** [comment]
@Carreau Chompy would be a nice addition to the game

**Comment by bollwyvl** [comment]
Or asteroids... shoot the kernel logos!

**Comment by damianavila** [comment]
> Or asteroids... shoot the kernel logos!

:100:

**Comment by bollwyvl** [comment]
...though it could be that the people that gave permission to use their
logos didn't mean it that way...

**Comment by rgbkrk** [comment]
![jupyter-asteroids](https://cloud.githubusercontent.com/assets/836375/10357736/3cfed9aa-6d4b-11e5-9ee6-9f8ae2b5122c.gif)

**Comment by willingc** [comment]
@rgbkrk You are my hero. Oddly this actually highlights the sponsors more than just the static logos.

**Comment by yuvipanda** [comment]
@rgbkrk Patch!

**Comment by jzf2101** [comment]
@Carreau is this issue still valid?

**Comment by Carreau** [comment]
Yes, I think it is, and could be tagged "help wanted". Alternatively we
could have that for April fools, and not only in nbviewer but Jupyter.org,
binder,...
